regolith/helpers/l_membershelper.py

```diff
@@ -67,14 +68,29 @@ def sout(self):
         bad_stati = ["finished", "cancelled", "paused", "back_burner"]
         people = []
         for person in self.gtx["people"]:
-            if rc.current and not person.get('active'):
-                continue
-            people.append(person)
+            if rc.current:
+                if not person.get('active'):
+                    continue
+                people.append(person)
+            elif rc.former:
+                if person.get('active'):
+                    continue
+                people.append(person)
+            else:
+                people.append(person)
+
         #people.sort()
```

Comment:
do you need help with this?  It is a bit tricky involving lambda functions.  You can try and google it or seek help.

Reply:
@sbillinge we have been working with Zach I will ask him and google it but if you have any other suggestions about contacting someone in the group who is familiar with this please let me know!